react/frontpage/components/ShowBlurb.scss

```diff
@@ -4,9 +4,17 @@ ShowsBlurb.scss
 Sass styling for show blurb on shows page (with grid)
 **/
 
+$mixcloud: #273a4b;
+$soundcloud: #ff8800;
+$tumblr: #35465c;
+$facebook: #3b5998;
```

Comment:
Since we're also using these variables in [`ShowPage.scss`](https://github.com/uclaradio/uclaradio/blob/master/react/frontpage/components/ShowPage.scss), can we move both of these to a single file? [`_common.scss`](https://github.com/uclaradio/uclaradio/blob/master/react/frontpage/_common.scss) might be a good location?